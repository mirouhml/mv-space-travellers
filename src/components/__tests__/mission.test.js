import React from 'react';
import renderer from 'react-test-renderer';
import Mission from '../Mission';

describe('OneMission', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Mission
        joined={false}
        id="123"
        name="name"
        description="desc"
      />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
