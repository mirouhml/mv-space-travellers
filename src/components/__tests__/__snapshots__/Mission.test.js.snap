// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission renders correctly 1`] = `
Array [
 <tr>
    <td>description</td>
    <td>
        <button className="member-btn active-member">
            Active Member
        </button>
    </td> 
     <td>
        <button className="member-btn non-member">
            Not a Member
        </button>
    </td>
      <td>
        <button className="mission-btn join-mission">
            Join Mission
        </button>
    </td>
      <td>
        <button className="mission-btn leave-mission">
            Leave Mission
        </button>
    </td>

 </tr>
]
`;

exports[`OneMission renders correctly 1`] = `
<tr>
  <td
    className="bold-text"
  >
    name
  </td>
  <td>
    desc
  </td>
  <td>
    <button
      className="member-btn non-member"
      type="submit"
    >
      Not a Member
    </button>
  </td>
  <td>
    <button
      className="mission-btn join-mission"
      onClick={[Function]}
      type="submit"
    >
      Join Mission
    </button>
  </td>
</tr>
`;
